cluster/src/main/java/com/hartwig/pipeline/execution/vm/VirtualMachineJobDefinition.java

```diff
@@ -78,6 +78,14 @@ static VirtualMachineJobDefinition germlineCalling(BashStartupScript startupScri
                 .build();
     }
 
+    static VirtualMachineJobDefinition sageCalling(BashStartupScript startupScript, ResultsDirectory resultsDirectory) {
+        return ImmutableVirtualMachineJobDefinition.builder()
```

Comment:
FYI i guess you've figured it out, but this will give you the default VM of 32 CPU and 120GB RAM. You can use VirtualMachinePerformanceProfile.custom() to tune that

Reply:
Yup. The default works nicely for me. I have got some profiling information here: https://github.com/hartwigmedical/hmftools/blob/master/sage/README2.md#performance-characteristics